ghost/i18n/locales/vi/portal.json

```diff
@@ -1,148 +1,148 @@
-    "You've successfully signed in.": "",
+    "Unsubscribed": "Đã hủy theo dõi",
+    "Unsubscribing from emails will not cancel your paid subscription to {{title}}": "Việc hủy theo dõi qua email sẽ không hủy gói đăng ký trả phí của bạn đối với {{title}}",
+    "Update": "Cập nhật",
+    "Update your preferences": "Cập nhật thiết lập",
+    "Verify your email address is correct": "Xác minh địa chỉ mail của bạn là đúng",
+    "View plans": "Xem các gói",
+    "We couldn't unsubscribe you as the email address was not found. Please contact the site owner.": "Chúng tôi không thể hủy theo dõi vì không tìm thấy địa chỉ email. Vui lòng liên hệ với chủ sở hữu trang web.",
+    "Welcome back, {{name}}!": "Chào mừng trở lại, {{name}}!",
+    "Welcome back!": "Chào mừng trở lại!",
+    "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "Khi hộp thư đến không nhận được email, nó thường được gọi là email bị trả lại. Điều này có thể là tạm thời. Tuy nhiên, trong một số trường hợp, email bị trả lại có thể là lỗi vĩnh viễn nếu địa chỉ email không hợp lệ hoặc không tồn tại.",
+    "Why has my email been disabled?": "Tại sao email của tôi bị vô hiệu hóa?",
+    "Yearly": "Hàng năm",
+    "You currently have a free membership, upgrade to a paid subscription for full access.": "Bạn đang là thành viên thường, hãy nâng cấp gói trả phí để có toàn quyền truy cập.",
+    "You have been successfully resubscribed": "Bạn đã theo dõi lại thành công",
+    "You're currently not receiving emails": "Những email bạn không nhận được gần đây",
+    "You're not receiving emails": "Bạn không nhận được email",
+    "You're not receiving emails because you either marked a recent message as spam, or because messages could not be delivered to your provided email address.": "Bạn không nhận được email vì bạn đã đánh dấu một email gần đây là thư rác, hoặc vì không thể gửi đến địa chỉ email bạn đã cung cấp.",
+    "You've successfully signed in.": "Bạn đã đăng nhập.",
     "Your account": "Tài khoản của bạn",
     "Your input helps shape what gets published.": "Thông tin của bạn giúp định hình nội dung được xuất bản.",
-    "Your subscription will expire on {{expiryDate}}": "",
-    "Your subscription will renew on {{renewalDate}}": "",
-    "Your subscription will start on {{subscriptionStart}}": ""
+    "Your subscription will expire on {{expiryDate}}": "Gói của bạn sẽ hết hạn vào {{expiryDate}}",
+    "Your subscription will renew on {{renewalDate}}": "Gói của bạn sẽ tự động gia hạn vào {{renewalDate}}",
+    "Your subscription will start on {{subscriptionStart}}": "Gói của bạn bắt đầu có hiệu lực vào {{subscriptionStart}}",
```

Comment:
Can you remove the trailing comma from here?

Reply:
I removed it.